### PR TITLE
Wait some time to make steps pass through.

### PR DESCRIPTION
**Pull request by barleyer**
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/91415/console

@xingxingxia  pls take a look,thanks!

### CODE REVIEW

File: features/cli/policy.feature [hunk 1]
```diff
@@ -60,10 +60,13 @@ Feature: change the policy of user/service account
 
     Given I have a project
     And I switch to the second user
+    And I wait up to 100 seconds for the steps to pass:
+    """
     When I run the :get client command with:
       | resource | project |
     Then the output should match:
```
Comment (pruan-rht) [inline features/cli/policy.feature:67]: we can replace 65-68 with `And the expression should be true> project(project.name).active?`

Comment (barleyer) [reply features/cli/policy.feature]: Good point, thanks! New logs: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/92738/console

Comment (pruan-rht) [reply features/cli/policy.feature]: @barleyer thanks!  
/lgtm

**Pull request merged**